src/components/CommentFooter/CommentFooter.less

```diff
@@ -16,6 +16,21 @@
     margin: 0 8px;
   }
 
+  &__bullet_br:after {
+    content: "\2022";
+    font-size: 14px;
+    margin: 0 8px;
+	  display: block;
+	  visibility: hidden;
+  	line-height: 0.5;
+	
+	  @media only screen and (min-width: 468px){
```

Comment:
Should probably used a predefined media query in `busy/src/styles/modules/variables.less`
or create one